### PR TITLE
feat(arc-495): add route for redirecting to hetarchief ssum register page

### PR DESCRIPTION
**Pull request by bertyhell**
image

image

### CODE REVIEW

File: src/config/index.ts [hunk 1]
```diff
@@ -54,6 +54,7 @@ const config = (): Configuration => {
 		cookieMaxAge: parseInt(getEnvValue('COOKIE_MAX_AGE', true), 10),
 		redisConnectionString: getEnvValue('REDIS_CONNECTION_STRING', false),
 		elasticSearchUrl: getEnvValue('ELASTICSEARCH_URL', true),
+		ssumRegistrationPage: getEnvValue('SSUM_REGISTRATION_PAGE', true),
```
Comment (bartnaessens) [inline src/config/index.ts:57]: Add ENV var for all ENVs (INT, TST, QAS, PRD) on ibm cloud

File: src/modules/auth/controllers/het-archief.controller.ts [hunk 2]
```diff
@@ -68,6 +71,36 @@ export class HetArchiefController {
...
 	}
 
+	@Get('register')
+	@Redirect()
+	public async registerRoute(
+		@Session() session: Record<string, any>,
+		@Query('returnToUrl') returnToUrl: string
+	) {
+		try {
+			const serverRedirectUrl = `${getConfig(
```
Comment (bartnaessens) [inline src/modules/auth/controllers/het-archief.controller.ts:81]: Why not build both using queryString.stringifyUrl ?

File: src/config/index.ts (hunk 1 above)
Comment (bertyhell) [reply src/config/index.ts:57]: done

File: src/modules/auth/controllers/het-archief.controller.ts [hunk 3]
```diff
@@ -15,7 +15,7 @@ import { ConfigService } from '@nestjs/config';
 import { ApiTags } from '@nestjs/swagger';
 import { Request } from 'express';
 import { get, isEqual, pick } from 'lodash';
-import queryString from 'query-string';
+import queryString, { stringifyUrl } from 'query-string';
```
Comment (bartnaessens) [inline src/modules/auth/controllers/het-archief.controller.ts:18]: Used for same purpose, so pick only 1

**Review by bartnaessens — APPROVED** [review]

**Pull request merged**